sdks/sdk-core/src/addresses.ts

```diff
@@ -292,6 +294,16 @@ const ASTROCHAIN_SEPOLIA_ADDRESSES: ChainAddresses = {
   v4QuoterAddress: '0x2cfa87651D3AB05Bc59E325E5eaC8495CF34cE55',
 }
 
+const UNICHAIN_ADDRESSES: ChainAddresses = {
+  v3CoreFactoryAddress: '0x1f98400000000000000000000000000000000003',
+  multicallAddress: '0xb7610f9b733e7d45184be3a1bc966960ccc54f0b',
```

Comment:
multical address is unichain sepolia address. looks like we haven't deployed [deploy-v3](https://github.com/Uniswap/deploy-v3)

Reply:
im wrong, they are from mainnet